docker-compose.yml

```diff
@@ -0,0 +1,41 @@
+version: "3"
+
+services:
+  mosaic:
+    build: .
+    environment:
+      - OAM_LAYER_ID=openaerialmap
+      - PORT=8001
+      - PGHOST=postgres
+      - PGUSER=postgres
+      - PGDATABASE=postgres
+      - PGPASSWORD=postgres
```

Comment:
is it normal, to store passwords this way?

Reply:
Local dev mode

Reply:
ok